src/mono/browser/runtime/invoke-cs.ts

```diff
@@ -15,12 +15,8 @@ import {
 } from "./marshal";
 import { mono_wasm_new_external_root, mono_wasm_new_root } from "./roots";
 import { monoStringToString } from "./strings";
-<<<<<<< HEAD
 import { utf16ToString } from "./strings";
```

Comment:
Group this import together with `import NativeAOT from "consts:nativeAOT";` (and add a whitespace) for future conflict prevention?

Reply:
Thanks, done.